clients/tool_results/lib/google_api/tool_results/v1beta3/connection.ex

```diff
@@ -12,20 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# NOTE: This class is auto generated by the swagger code generator program.
-# https://github.com/swagger-api/swagger-codegen.git
+# NOTE: This class is auto generated by the elixir code generator program.
 # Do not edit the class manually.
 
 defmodule GoogleApi.ToolResults.V1beta3.Connection do
   @moduledoc """
   Handle Tesla connections for GoogleApi.ToolResults.V1beta3.
   """
 
+  @type t :: Tesla.Env.client()
+
   use GoogleApi.Gax.Connection,
     scopes: [
       # View and manage your data across Google Cloud Platform services
       "https://www.googleapis.com/auth/cloud-platform"
     ],
-    otp_app: :google_api_tool_results,
-    base_url: "https://www.googleapis.com/toolresults/v1beta3/projects"
+    otp_app: :google_api_toolresults,
```

Comment:
@chingor13 This is used as the config key for the base url (and presumably other configs). I guess I don't have a strong opinion on whether it should be the API name (which generally won't have underscores) or the snake-ization of the Elixir module name (which might). Do you have thoughts on this?

Reply:
I think we should prefer the earlier version. Opened https://github.com/googleapis/elixir-google-api/pull/1846